core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1298,6 +1328,8 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
 
       _ = validateSubmissionRootEntity(submissionRequest, methodConfig)
 
+      _ = submissionRequest.userComment.map(validateMaxStringLength(_, "userComment", 1000))
```

Comment:
Where is the escaping of the user-supplied string done?

Reply:
I know it's only used twice, but should 1000 be a constant? I worry that there is no documentation about the limit.

Reply:
While I was testing, I realized that both Terra UI and Swagger API does the necessary escaping before sending the request to Rawls. Also I think, the unmarshaller that converts the request to proper request case classes covers the escaping. Hence I didn't add any additional escaping here. But I would appreciate if someone else tested this as well. If I missed something, I can add additional escaping.